awx/api/urls.py

```diff
@@ -298,68 +308,68 @@ def url(regex, view, kwargs=None, name=None, prefix=''):
     url(r'^$',                                          'notification_template_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'notification_template_detail'),
     url(r'^(?P<pk>[0-9]+)/test/$',                      'notification_template_test'),
     url(r'^(?P<pk>[0-9]+)/notifications/$',             'notification_template_notification_list'),
-)
+]
 
-notification_urls = patterns('awx.api.views',
+notification_urls = [
     url(r'^$',                                          'notification_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'notification_detail'),
-)
+]
 
-label_urls = patterns('awx.api.views',
+label_urls = [
     url(r'^$',                                          'label_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'label_detail'),
-)
+]
 
-workflow_job_template_node_urls = patterns('awx.api.views',
+workflow_job_template_node_urls = [
     url(r'^$',                                          'workflow_job_template_node_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'workflow_job_template_node_detail'),
     url(r'^(?P<pk>[0-9]+)/success_nodes/$',             'workflow_job_template_node_success_nodes_list'),
     url(r'^(?P<pk>[0-9]+)/failure_nodes/$',             'workflow_job_template_node_failure_nodes_list'),
     url(r'^(?P<pk>[0-9]+)/always_nodes/$',              'workflow_job_template_node_always_nodes_list'),
-)
+]
 
-workflow_job_node_urls = patterns('awx.api.views',
+workflow_job_node_urls = [
     url(r'^$',                                          'workflow_job_node_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'workflow_job_node_detail'),
     url(r'^(?P<pk>[0-9]+)/success_nodes/$',             'workflow_job_node_success_nodes_list'),
     url(r'^(?P<pk>[0-9]+)/failure_nodes/$',             'workflow_job_node_failure_nodes_list'),
     url(r'^(?P<pk>[0-9]+)/always_nodes/$',              'workflow_job_node_always_nodes_list'),
-)
+]
 
-schedule_urls = patterns('awx.api.views',
+schedule_urls = [
     url(r'^$',                                          'schedule_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'schedule_detail'),
     url(r'^(?P<pk>[0-9]+)/jobs/$',                      'schedule_unified_jobs_list'),
-)
+]
 
-activity_stream_urls = patterns('awx.api.views',
+activity_stream_urls = [
     url(r'^$',                                          'activity_stream_list'),
     url(r'^(?P<pk>[0-9]+)/$',                           'activity_stream_detail'),
-)
+]
 
-instance_urls = patterns('awx.api.views',
-    url(r'^$',                                          'instance_list'),
-    url(r'^(?P<pk>[0-9]+)/$',                           'instance_detail'),
-    url(r'^(?P<pk>[0-9]+)/jobs/$',                      'instance_unified_jobs_list'),
-    url(r'^(?P<pk>[0-9]+)/instance_groups/$',           'instance_instance_groups_list'),
-)
+instance_urls = [ 
+    url(r'^$', 'instance_list'),
```

Comment:
nitpick but I noticed this whitespace padding is different than the others.

I actually like *this* rather than the artificial padding that we use elsewhere (as does my editor).